authors/apps/authentication/tests.py

```diff
@@ -262,37 +254,55 @@ class UserRetrieveUpdateAPIViewTestCase(TestCase):
 
     def setUp(self):
 
-        self.existing_user_data = {
-            "username": "janejones",
-            "email": "jjones@email.com",
+        self.user_data = {
+            "username": "janejones2",
+            "email": "jjones@email2.com",
             "password": "Enter-123"}
-
-        self.existing_user = User.objects.create_user(
-            **self.existing_user_data)
+        self.activate_url = reverse('authentication:activate-account')
+        self.register_url = reverse('authentication:register')
+        self.login_url = reverse('authentication:login')
+        self.user_action_url = reverse('authentication:user-action')
         self.client = APIClient()
+
         response = self.client.post(
-            '/api/users/login/',
-            {"user": {"email": "jjones@email.com", "password": "Enter-123"}},
-            format='json'
-        )
-        self.token = response.data["token"]
+            self.register_url, {"user": self.user_data}, format="json")
+
+        self.token = response.data["data"]["token"]
+
+        self.client.get(f"{self.activate_url}?token={self.token}", format='json')
+
+        response = self.client.post(
+            self.login_url, {"user": self.user_data}, format='json')
+
         self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
 
+    def test_verify_account_with_invalid_token(self):
+        token = "icVpZnJlZCIsImV4cCI6MTU1ODY3"
```

Comment:
Thanks @RonKbS, but that token was used to test invalid token meaning am not using it anywhere as private data. In other words, it could be any string that's why I didn't store it in environment variables. How do you see?

Reply:
Alright then. 
Please rename this variable to invalid_token. Same for line 289

Reply:
Thanks for your feedback. Let me work on it